### PR TITLE
Recognize annotations made by an #ask transcluded template

### PR DESCRIPTION
**Pull request by mwjames**
refs #710, After looking at  smw.org@`Template:Invert-property` on how #ask is used in connection with templates to generate/annotate invert property annotations, this PR will allow to use a more simplified approach.

Use `{{InvertPropertyFinder|Located in|Location of}}` 

`InvertPropertyFinder`
```
<includeonly>{{#ask:[[{{{1}}}::{{PAGENAME}}]]|link=none|sep=|template=InvertPropertySetter|userparam={{{2}}} }}</includeonly>
```
`InvertPropertySetter`
```
<includeonly>{{#set:|{{{userparam}}}={{{1}}} }}</includeonly>
```

### CODE REVIEW

**Comment by mwjames** [comment]
@kghbln An example on how templates and `#ask` can be used more efficiently together.

File: tests/phpunit/Integration/MediaWiki/AskParserFunctionDBIntegrationTest.php [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+		$this->pageCreator
+			->createPage( Title::newFromText( __METHOD__ . '00' ) )
+			->doEdit( '{{#set:|TestPropertyByAskTemplate=TestValueByAskTemplate}}' );
+
+		$this->pageCreator
+			->createPage( $subject->getTitle() )
+			->doEdit(
+				'{{#ask:[[TestPropertyByAskTemplate::TestValueByAskTemplate]]|link=none|sep=|template=AskTemplateToAddPropertyAnnotation}}' );
+
+		$expected =	array(
```
Comment (JeroenDeDauw) [inline tests/phpunit/Integration/MediaWiki/AskParserFunctionDBIntegrationTest.php]: tab indent?

**Comment by JeroenDeDauw** [comment]
Code looks ok, though I do not fully understand the context without digging deeper

**Comment by mwjames** [comment]
https://semantic-mediawiki.org/wiki/Examples/Parser/Ask_with_template_to_find_invert_property

**Pull request merged**